docs/data/sdks/typescript-node/index.md

```diff
@@ -0,0 +1,363 @@
+---
+title: Typescript Node.js
+description: The Amplitude Typescript SDK Installation & Quick Start guide.
+icon: material/language-typescript
+---
+
+
+![npm version](https://badge.fury.io/js/@amplitude%2Fanalytics-node.svg)
+
+!!!info "SDK Resources"
+    - [TypeScript Node SDK Reference :material-book:](https://amplitude.github.io/Amplitude-TypeScript/)
+    - [TypeScript Node SDK Repository :material-github:](https://github.com/amplitude/Amplitude-TypeScript/tree/main/packages/analytics-node)
+    - [TypeScript Node SDK Releases :material-code-tags-check:](https://github.com/amplitude/Amplitude-TypeScript/releases)
+
+The TypeScript Node.js SDK lets you send events to Amplitude. This library is open-source, check it out on [GitHub](https://github.com/amplitude/Amplitude-TypeScript).
+
```

Comment:
We should probably add a warning here that codegen (Ampli) is not available for this runtime.